static/bidder-info/ownadx.yaml

```diff
@@ -0,0 +1,12 @@
+endpoint: "http://{{.Host}}-prebid.owadx.com/bidder/bid/{{.AccountID}}/000?token={{.SourceId}}"
```

Comment:
still doesn't fit criteria of `bidders with dynamic hosts` https://github.com/prebid/prebid-server/issues/2612

will follow up on team on this